tsl/src/compression/api.c

```diff
@@ -44,6 +45,9 @@
 #include "scanner.h"
 #include "scan_iterator.h"
 #include "utils.h"
+#include "chunk.h"
+#include "custom_type_cache.h" // for the compressed data datatype
+#include <utils/snapmgr.h>
```

Comment:
Please consider moving this to the group of other `utils` headers (i.e., line 27).